### PR TITLE
draft code of conduct

### PR DESCRIPTION
**Pull request by bast**
- missing: contact information and procedure for handling incidents (search for "warning" for text we need to adapt before merging)

- part of #270

### CODE REVIEW

**Comment by bast** [comment]
Suggestions welcome on how to handle incidents. Without contact information and a clear process, the code of conduct is incomplete.

**Comment by rdoursenaud** [comment]
For the contact, I think that the @mido handle should be enough. It notifies all the repository admins.
As for the process I don't have any inputs except have a provision for some "right to make mistakes". We should first try to discuss with the problematic user and convey why their behavior was inappropriate before using any [banning tool](https://docs.github.com/fr/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-organization).

**Comment by bast** [comment]
Good idea. I will adjust the text.

**Comment by bast** [comment]
I have also added our email addresses (publicly available on GitHub) since persons might prefer to report incidents privately via email.

**Review by rdoursenaud — COMMENTED** [review]
I forgot to submit my review. Here it is!

File: docs/code-of-conduct.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+Scope
+-----
+
+This Code of Conduct applies to the following online spaces:
+
+-  Code repositories, issue trackers, and pull requests made against
+   the Mido GitHub organization.
+-  Discussions in the Mido GitHub organization.
+-  The Mido mailing list.
```
[the remark this answers is not in the document]
Comment (bast) [reply docs/code-of-conduct.rst:103]: I will check.

**Comment by bast** [comment]
Thanks for good comments. I will address these.

File: docs/code-of-conduct.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (bast) [reply docs/code-of-conduct.rst:103]: You are right. It does not exist anymore.

**Review by rdoursenaud — APPROVED** [review]
Nice!

**Pull request merged**